companion/src/radiodata.h

```diff
@@ -1213,7 +1217,7 @@ class GeneralSettings {
     unsigned int backlightColor;
     CustomFunctionData customFn[CPN_MAX_CUSTOM_FUNCTIONS];
     char switchName[18][3+1];
-    unsigned int switchConfig[18];
+    unsigned int switchConfig[CPN_MAX_SWITCHES];
```

Comment:
`getBoardCapability()`, `getCurrentFirmware()`, `g`, `current_firmware_variant`, etc...  Quite a few in eeprominterface.h, it seems like.